config/salt/minion.d-ca/signing_policies.conf

```diff
@@ -3,8 +3,8 @@ x509_signing_policies:
     - minions: '*'
     - signing_private_key: /etc/pki/private/ca.key
     - signing_cert: /etc/pki/ca.crt
-    - basicConstraints: "CA:false"
-    - keyUsage: "nonRepudiation, digitalSignature, keyEncipherment"
+    - basicConstraints: "critical CA:false"
+    - keyUsage: "critical nonRepudiation, digitalSignature, keyEncipherment"
```

Comment:
Yeah, I have been wondering for a long time if it wasn't just a typo on the documentation... anyways, thanks!